fiftyone/core/dataset.py

```diff
@@ -3137,14 +3137,13 @@ def save_view(
             raise ValueError("Cannot save view into a different dataset")
 
         view._set_name(name)
-        slug = self._validate_saved_view_name(name, overwrite=overwrite)
+        self._validate_saved_view_name(name, overwrite=overwrite)
```

Comment:
why are we removing slug?

Reply:
I don't see the point of slug if it is not something made apparent to the user? What is the benefit?